packages/neuron-ui/src/widgets/Dialog/dialog.module.scss

```diff
@@ -50,6 +50,10 @@
   position: relative;
   max-height: calc(100vh - 260px);
   overflow-y: auto;
+
+  &::-webkit-scrollbar {
+    width: 0;
```

Comment:
Should all pop-up scrollbars be set to 0? If so, wouldn't it be impossible to scroll using the mouse wheel?